draft-ietf-cbor-7049bis.md

```diff
@@ -2072,15 +2072,14 @@ appropriate resource management to mitigate these attacks.  (Items for
 which very large sizes are given can also attempt to exploit integer
 overflow vulnerabilities.)
 
-Applications where a CBOR data item is examined by a gatekeeper
-function and later used by a different application may exhibit
-vulnerabilities when multiple interpretations of the data item are
-possible. For example, an attacker could make use of duplicate keys in
-maps and precision issues in numbers to make the gatekeeper base its
+Protocols that are used in a security
+context should be defined in such a way that multiple
+interpretations are reliably reduced to a single one. 
```

Comment:
At this point "multiple interpretations" hasn't been introduced yet.  Maybe just say "multiple potential interpretations".

Reply:
That works too.